demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/10/Version20241004090021.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Exception;
+use Doctrine\DBAL\Platforms\MySQLPlatform;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+class Version20241004090021 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'refs T: adjust column category length';
```

Comment:
Please add the issue number

Reply:
done in https://github.com/demos-europe/demosplan-core/pull/3763/commits/8d653e0027ccf43fc20f94feb528e2a6977cfb3d thx